### ISSUE TITLE
PluralSight Angular Fundamentions EventApp not working per course instructions

### ISSUE THREAD

**Issue by jrmcdona** [joeeames/AngularFundamentalsFiles#13]
Hello,

This is simply all I can get to render on the EventDetails.html page. 
{{event.name}}

I have followed the PluralSight course and created EventController.js and just cannot the event.name variable to populate.  Looks dead simple but just isn't working.

app.js
'use strict';
var eventsApp = angular.module('eventsApp', []);

EventDetails.html
<head lang="en" ng-app="eventsApp">

<div class="container">
    <div ng-controller="EventController">
        {{event.name}}
    </div>
</div>

<script src="lib/jquery.min.js"></script>
<script src="lib/underscore-1.4.4.min.js"></script>
<script src="lib/bootstrap.min.js"></script>
<script src="lib/angular/angular.js"></script>
<script src="js/app.js"></script>
<script src="js/controllers/EventController.js"></script>

EventController.js
'use strict'
eventsApp.controller('EventController',
    function EventController($scope){
        $scope.event = {
            name: 'Angular Boot Camp'
        };
    }
);

What am I missing?  I have gone back over the course many times but just cannot figure it out.   The page must think {{event.name}} is just text I am writing out.  

Thanks

**Comment by jrmcdona** [comment]
Update: using a JavaScript function works.   Curious to why it doesn't work the others way I stated above?

This works:
function EventController($scope) {
    $scope.event = {
        name: 'Angular Boot Camp'
    };
}

**Comment by jrmcdona** [comment]
Sheesh.  Resolved.

ng-app="eventsApp"  was in the <head> instead of the <html>.

**Comment by jrmcdona** [comment]
Also, you can not have this function inside the other function.

I think the Training is out of date.

This below will not work, but this is how they do in the training.

eventsApp.controller('EventController',
function EventController($scope){
$scope.event = {
name: 'Angular Boot Camp'
};
}
);

**Comment by joeeames** [comment]
no idea why you say that jrmcdona.  That works just fine.  Please see this plunk: http://plnkr.co/edit/hylrny7inYZmbgJoM6rg?p=preview

**Comment by jrmcdona** [comment]
No clue either.  I spent hours trying to make it work thinking it was the only way.   After trial and error I found a mistake in my ng-app tage and got it to work using code from stackoverflow,  then I put the course controller code back in and it failed again.  I compared the two and the difference was the function.   I took that out and I am up and running.  Will try again with the function tomorrow.
